utils/imageSaver.py

```diff
@@ -29,22 +30,19 @@ def save_image():
         }
     ]
 
-    answer = prompt(questions)["download_type"]
+    answer: Dict = prompt(questions)["download_type"]
 
     if answer != 'Path':
-        if answer == 'TID Repo':
-            image_data_url = "https://raw.githubusercontent.com/Muhimen123/TID/main/image_data.json"
-            link = tid_repo_prompt(image_data_url)
+        link: Optional[str] = tid_repo_prompt(
+            "https://raw.githubusercontent.com/Muhimen123/TID/main/image_data.json"
+        ) if answer == 'TID Repo' else input("Please enter download link: ")
 
-        else:
-            link = input("Please enter download link: ")
-
-        path = os.path.join(os.getcwd(), "data", "images")
+        path: str = os.path.join(os.getcwd(), "data", "images")
         downloader.download_image(link, path, "current_desktop_wallpaper.png")
```

Comment:
Instead of downloading `current_desktop_wallpaper.png` in `images` folder, it's saving it in a file name like `imagescurrent_desktop_wallpaper.png`. It's because the path doesn't end with `//`. Prefixing the `current_desktop_wallpaper.png` with `//` should work.

Reply:
fixed !